tests/data/tasks/profile-to-osco/output-set/profile.yaml

```diff
@@ -0,0 +1,172 @@
+apiVersion: api_version
```

Comment:
Since I'm right now working with profiles and catalogs, I'd prefer not to have something called profile.yaml that isn't an oscal profile.  It might cause confusion somehow in users.  So I'd rather it had a more explicit name.  profile_spec.yaml?

Reply:
second on this.